### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by GHarte**

### CODE REVIEW

File: docs/android/widgets/simpleWidget.md [hunk 1]
```diff
@@ -8,8 +8,11 @@ permalink: /docs/android/widgets/simple/
 ---
 
 # Simple Widget
-
 {: .no_toc }
+<small>cartrawler.core.ui.views.partner.CTSimpleWidget</small>
+
```
Comment (firminofranciele007) [inline docs/android/widgets/simpleWidget.md:13]: Maybe this one could be highlighted a bit more cause it is the main thing we need when adding the widget
"cartrawler.core.ui.views.partner.CTSimpleWidget" just make the font darker and bigger would help

File: docs/android/widgets/vehicleWidget.md [hunk 2]
```diff
@@ -8,13 +8,18 @@ permalink: /docs/android/widgets/vehicle/
 ---
 
 # Vehicle Widget
-
 {: .no_toc }
 
+<small>cartrawler.core.ui.views.partner.CTVehicleWidget</small>
```
Comment (firminofranciele007) [inline docs/android/widgets/vehicleWidget.md:13]: Same here for highlight

File: docs/react native/androidIntegration.md [hunk 3]
```diff
@@ -8,19 +8,37 @@ permalink: docs/react-native/android-integration/
 ---
 
 # Android Integration
-
 {: .no_toc }
 
 To integrate the Android SDK's within your React Native app, please use the following steps.
```
Comment (firminofranciele007) [inline docs/react native/androidIntegration.md:13]: We could reword it if removing the section below "To integrate the Android SDK's within your React Native app, open the Android folder inside your React Native project using Android Studio and follow the steps listed below."

File: docs/android/widgets/simpleWidget.md (hunk 1 above)
Comment (GHarte) [reply docs/android/widgets/simpleWidget.md:13]: Make it bold and or remove the <small> tags?

Comment (firminofranciele007) [reply docs/android/widgets/simpleWidget.md:13]: Yeah I think that should be enough, cause I nearly missed it when looking at it cause it is really light grey

**Pull request merged**